### PR TITLE
Test with kitchen-dokken and be explicit about the required Chef version

### PR DESCRIPTION
**Pull request by tas50**
Test in travis with kitchen-dokken and the latest chef/chef docker image.

### CODE REVIEW

File: .kitchen.docker.yml [hunk 1]
```diff
@@ -1,21 +1,97 @@
 driver:
-  name: docker
+  name: dokken
+  privileged: true # because Docker and SystemD/Upstart
+  chef_image: chef/chef
+  chef_version: <%= ENV['CHEF_VERSION'] || '12.5.1' %>
```
Comment (tduffield) [inline .kitchen.docker.yml]: Why 12.5.1?

Comment (scotthain) [inline .kitchen.docker.yml]: IIRC 12.5.1 is the 'earliest' the cookbook supports. I don't remember the functionality that was introduced in 12.5.1 but it was something...

Comment (tas50) [inline .kitchen.docker.yml]: custom resources?

**Comment by tas50** [comment]
Assuming the tests all pass this should be good for a review @tduffield @schisamo @scotthain

**Pull request merged**